src/main/java/com/griefcraft/modules/pluginsupport/Factions.java

```diff
@@ -99,4 +102,43 @@ public boolean canProtectLegacy(FPlayer fPlayer, Faction faction) throws Illegal
 
         return false;
     }
+
+    @Override
+    public void onAccessRequest(LWCAccessEvent event) {
+        Player player = event.getPlayer();
+        com.griefcraft.model.Protection protection = event.getProtection();
```

Comment:
should import

Reply:
Can't import because there is already a Protection class imported by the Factions plugin. 
line 11: `import com.massivecraft.factions.config.file.MainConfig.Factions.Protection;`

Reply:
I see, that's a bit annoying. Oh well, nothing you can do about it then.